### DISCUSSION TITLE
HELP HELP HELP

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#632] [Q&A] [answered]
Is there anyone out there of the plug team that can help me publish hundreds of meetings in one go????

I was going to refresh my database today and have to revert back by restoring the original postings and they come back in DRAFTR and I need them published without having to navigate through the list one by one.

**Comment by OSAwebsite** [comment]
Well, it took me 1.5 hours but it is done. Back to how it was before the refreshing of the database. I will write more about this in a separate discussion, because the import sheet urgently needs to be looked at. I will do some testing and report my findings.

One of the things I find confusing is that 'bugs' are only to be entered by developers. But in 'Discussions' there is no 'bug' label.  I think we should have that label and then the team can decide whether they deem it a real 'bug' of importance to the development team or revert it into a discussion.

**Comment by OSAwebsite** [comment]
Excuse my brevity: this failed change has eaten up my whole day and I am annoyed - with myself largely, because I should not have trusted the plugin import to work without issues. I should have done it on my staging site. Lessons learned.

**Comment by brianw-area24** [comment]
It sounds like you're trying to import a CSV file, correct?  Please get us a copy of that file, either post it here, or email it to devs@code4recovery.org.

**Comment by OSAwebsite** [comment]
See my recent posting in the Bugs section... has the upload and other
documentation


-
Yours in service,
Nicky D
Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*

**Comment by brianw-area24** [comment, marked as answer]
Duplicate of Issue #633, see discussion there
